client/store/items.ts

```diff
@@ -30,23 +33,47 @@ export const getters: GetterTree<ItemsState, ItemsState> = {
 
 export const mutations: MutationTree<ItemsState> = {
   ADD_ITEM(state: ItemsState, { item }: { item: BaseItemDto }) {
+    console.time('ADD_ITEM');
```

Comment:
Is that necessary to keep?

Reply:
It was there for @ferferga to test and compare timings. The PR hasn't been touched by me since :yum: